camunda-formio-tasklist-vue/docs/components/TaskList.md

```diff
@@ -19,6 +19,7 @@ solution in your Vue.js based project.
 | containerHeight | string | 100vh | Prop to adjust the height values of Vue component, in case of Vue component is too much for your integrated application. You can adjust the height from the range 100-400 |
 | taskSortBy | string | "created" | Prop to decide by default what value should the tasks be sorted when displayed at first. You can use the values like - "created", "dueDate", "follweowUpDate", "priority", "name", "assignee" |
 | taskSortOrder  | string | "desc" | Prop to decide by default what value should the tasks should be ordered when displayed at first. You can use the values like - "desc", "asc" |
+| taskDefaultFilterListNames | Array[string] | optional | Prop to set default filterNames to be available for the component and corresponding to it get the filtered Tasklist. `eg: ['Clerk Tasks', 'XYZ |
```

Comment:
We would need options to hide the buttons too. By default all buttons like Filters, Forms are displayed.
Then we need props to hide the Forms (optionally).

Reply:
Yep will work on it in next week